monitoring/accessing-third-party-uis.adoc

```diff
@@ -12,7 +12,7 @@ Integrated Metrics, Alerting, and Dashboard UIs are provided in the {product-tit
 * xref:../monitoring/reviewing-monitoring-dashboards.adoc#reviewing-monitoring-dashboards[
 Reviewing monitoring dashboards]
 
-{product-title} also provides access to the Prometheus, Alertmanager, and Grafana third-party interfaces.
+{product-title} also provides access to the Prometheus, Alertmanager, and Grafana third-party interfaces. Additional platform components are included in *Monitoring* → *Dashboards* in the {product-title} web console.
```

Comment:
```suggestion
{product-title} also provides access to the Prometheus, Alertmanager, and Grafana third-party interfaces. Dashboards for some additional platform components are included in *Monitoring* → *Dashboards* in the {product-title} web console.
```

Reply:
Or something similar to clarify that the only additional thing on that page is some more dashboards.